modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -15580,6 +15580,8 @@ scim-generate-access-token=Generate Access Token
 scim-generate-access-token-help=Generate a new access token while establishing a 10 day grace period for previous access tokens before they are invalidated.
 scim-matcher-field=Matcher Field
 scim-matcher-field-help=The matcher field is used to match and associate existing users that are not already associated with a SCIM client. Valid options are schema properties from RFC 7643 that will be matched with appropriate user fields.
+scim-reset-client-data=Reset SCIM Client Provisioning Data
+scim-reset-client-data-help=All the current SCIM Client related data and the generated OAuth 2 tokens is removed. This is necessary for being able to configure a new SCIM Client.
```

Comment:
Small typo, it should be `...generated OAuth 2 tokens ARE removed.`

Just pointing this out FYI; I will take care of this.